carta/image.py

```diff
@@ -65,8 +69,15 @@ def new(cls, session, path, hdu, append):
         """
         path = session.resolve_file_path(path)
         directory, file_name = posixpath.split(path)
-        image_id = session.call_action("appendFile" if append else "openFile", directory, file_name, hdu, return_path="frameInfo.fileId")
-
+        command = "appendFile" if append else "openFile"
+        if complex is False:
+            image_arithmetic = False
+        elif complex is True:
+            image_arithmetic = "true"
+            if Expression is None:
+                Expression = "AMPLITUDE"
```

Comment:
This won't be necessary if `expression` defaults to amplitude in the two session methods and is required here.

Reply:
Done in f651be6.